src/renderer/getDiForUnitTesting.tsx

```diff
@@ -31,6 +33,12 @@ export const getDiForUnitTesting = ({ doGeneralOverrides } = { doGeneralOverride
   if (doGeneralOverrides) {
     di.override(getValueFromRegisteredChannelInjectable, () => () => undefined);
 
+    di.override(readDirInjectable, () => () => Promise.resolve([]));
```

Comment:
This override should throw. Check other comment related to `setup`-time asynchronicity for solution.